distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -296,8 +306,83 @@ public void testOpensearchToCswSourceToCswEndpointQuerywithMetacardXml() throws
         String queryUrl = OPENSEARCH_PATH + "?q=" + DEFAULT_KEYWORD + "&format=xml&src="
                 + CSW_SOURCE_WITH_METACARD_XML_ID;
 
-        when().get(queryUrl).then().log().all().assertThat().body(containsString(RECORD_TITLE_1),
-                containsString(RECORD_TITLE_2));
+        when().get(queryUrl).then().log().all().assertThat()
+                .body(containsString(RECORD_TITLE_1), containsString(RECORD_TITLE_2));
+    }
+
+    @Test
+    public void testListAllSourceInfo() {
+        try {
+            setupConnectedSources();
+        } catch (IOException e) {
+            LOGGER.error("Couldn't create connected sources: {}", e.getMessage());
+        }
+
+        String queryUrl = ADMIN_SOURCE_PATH + ADMIN_ALL_SOURCE_INFO_PATH;
+        given().auth().basic("admin", "admin").when().get(queryUrl).then().log().all().assertThat()
+                .body(containsString("\"fpid\":\"OpenSearchSource\""),
+                        containsString("\"fpid\":\"Csw_Federated_Source\""),
+                        containsString("\"fpid\":\"Csw_Connected_Source\""));
+    }
+
+    @Test
+    public void testFederatedSourceStatus() {
+        String sourcesURL = ADMIN_SOURCE_PATH + ADMIN_ALL_SOURCE_INFO_PATH;
```

Comment:
looks like you use these in two functions and they are static.  I would suggest making them final static constants on this class.